pkgs/by-name/fi/filterpath/package.nix

```
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  testers,
+  filterpath
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "filterpath";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Sigmanificient";
+    repo = "filterpath";
+    rev = finalAttrs.version;
+    hash = "sha256-vagIImWQQRigMYW12lw+Eg37JJ2yO/V5jq4wD3q4yy8=";
+  };
+
+  makeFlags = [
+    "CC=${stdenv.cc}/bin/cc"
```

```suggestion
    "CC=cc"
```

Otherwise you'll very likely break cross compilation.

---

I think it may be better to use `lib.getExe`

---

> I think it may be better to use `lib.getExe`

No. It's `stdenv.buildPlatform.cc`, which should already be provided in stdenv.